.eslintrc.js

```diff
@@ -77,5 +77,12 @@ module.exports = {
         '@typescript-eslint/no-unused-expressions': 'off',
       },
     },
+    {
+      files: ['**/test/typegen/**/*.ts'],
+      rules: {
+        'eslint-comments/no-unlimited-disable': 'off',
+        'eslint-comments/no-unused-disable': 'off',
```

Comment:
What's going on here?

The pattern doesn't satisfy any changed file.

Reply:
This is tied to the typegen-generated files that `eslint` is yelling about until I disable these rules. They aren't in the PR because I [ignored](https://github.com/FuelLabs/fuels-ts/pull/2123/files/1485d335c825dc5096f61100a03bb1f75918bc6c#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R148) them.